server/package.json

```diff
@@ -13,6 +13,7 @@
     "cors": "^2.8.5",
```

Comment:
Good thinking using a third-party library! This is way easier than coding our own ID function.